src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -186,7 +192,7 @@ export class RbdFormComponent extends CdForm implements OnInit {
         namespace: new UntypedFormControl(null),
         useDataPool: new UntypedFormControl(false),
         dataPool: new UntypedFormControl(null),
-        size: new UntypedFormControl(null, {
+        size: new UntypedFormControl(1, {
```

Comment:
const DEFAULT_SIZE = 1

```suggestion
        size: new UntypedFormControl(DEFAULT_SIZE, {
```